cmd/root/auth.go

```diff
@@ -152,7 +152,11 @@ func MustWorkspaceClient(cmd *cobra.Command, args []string) error {
 			return err
 		}
 		if b := bundle.GetOrNil(cmd.Context()); b != nil {
-			cfg = b.WorkspaceClient().Config
```

Comment:
This path is executed only once per command though, so that's fine.

But this fix doesn't fix the issue when using bundle commands. We need a mutator that verifies that the workspace client is valid and returns an error if it isn't. That mutator can make the same call.

Reply:
Done, added the mutator.